test/support.exs

```diff
@@ -24,6 +24,15 @@ defmodule Raxx.Forwarder do
   end
 end
 
+defmodule Raxx.HandleRequestForwarder do
```

Comment:
Probably this should be called `Raxx.RaiseException` or `Raxx.Kaboom!` depending on how you are feeling? curious question can module names also have `!` in them. Also I wouldn't bother with a specific path.

Reply:
I thought you could match on the path for different behaviours. So then you have a generic HandleRequestForwarder with different paths like `raise_error`, `render_ok` etc

Reply:
Raxx.Forwarder works for all the render_ok cases etc. Also if going off paths nothing about this module is a forwarder